veocean-data-farming/df-background.md

```diff
@@ -0,0 +1,55 @@
+---
+description: Intuition and background on the Data Farming program.
+---
+# Data Farming Background
+
```

Comment:
Add a line at the top to summarize the goal:

"Data Farming (DF) incentivizes for growth of Data Consume Volume (DCV) in the Ocean ecosystem."

Reply:
Done